stories/topics/proc-azure-monitor-deployment-engine.adoc

```diff
@@ -18,6 +18,13 @@ Approximately 10 minutes into the deployment process, the *Outputs* section of t
 . Login to the deployment using the following credentials:
   * *Username*: _admin_
   * *Password*: Use the _Administrator Password_ that you chose when configuring your deployment.
+. The deployment driver displays a message indicating that your deployment is underway.
+Click btn:[Log in with Red Hat account]. The **Red Hat login** page opens.
```

Comment:
Is this a separate step? Does it need a . ?

Reply:
Its not a separate step from user perspective, once you logs in, they will see the page where deployment is happening in the background and there is a modal dialog in the front.